x-pack/plugins/security_solution/common/endpoint/data_generators/exceptions_list_item_generator.ts

```diff
@@ -173,10 +169,33 @@ export class ExceptionsListItemGenerator extends BaseDataGenerator<ExceptionList
     };
   }
 
+  generateTrustedAppSignerEntry(): EntriesArray {
+    return [
+      {
+        field: 'process.Ext.code_signature',
```

Comment:
I've only realized this now as I went to test this. This should not include `Ext` in that path. macOS events do not use it. Obviously this comment applies throughout this PR.

@ricardoungureanu is `process.code_signature` available in all Endpoint releases since 7.17.0? Basically, will adding Trusted Apps by signer work for all 7.17.0+ Endpoints?

```suggestion
        field: 'process.code_signature',
```

Reply:
Updated, can be tested now.